### DISCUSSION TITLE
Browser blocks

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#6532] [Archive]
Guys, a little help, I have directus hosted locally on our servers, VM with 4 processors and 4048 Gb memory, 10Gb network adapter, but it happens with users who are working via VPN, especially when they change a data field, the browser blocks. Does anyone have a similar situation? or does anyone imagine what could be causing this situation?

Thanks.

**Comment by rijkvanzanten** [comment]
Hard to tell without some more specific information 🤔 It sounds like the problem lies on the front-end if a certain sequence of events causes just the browser to hang.

Do you have a step-by-step that triggers the problem consistently?
